model/common/tests/grid_tests/test_vertical.py

```diff
@@ -293,3 +293,37 @@ def test_vct_a_vct_b_calculation_from_icon_input(
 
     assert helpers.dallclose(vct_a.ndarray, grid_savepoint.vct_a().ndarray)
     assert helpers.dallclose(vct_b.ndarray, grid_savepoint.vct_b().ndarray)
+
+
+@pytest.mark.datatest
+@pytest.mark.parametrize("experiment", [dt_utils.GAUSS3D_EXPERIMENT])
+def test_init_vert_coord(
```

Comment:
In case you want to test for several backends you need to pass the `backend` test fixture.